web/packages/shared/components/ButtonTextWithAddIcon/ButtonTextWithAddIcon.tsx

```diff
@@ -0,0 +1,59 @@
+/**
+ * Teleport
+ * Copyright (C) 2023  Gravitational, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import React from 'react';
+import { ButtonText } from 'design';
+import { Add as AddIcon } from 'design/Icon';
+
+export const ButtonTextWithAddIcon = ({
+  label,
+  onClick,
+  disabled,
+  iconSize = 12,
+}: {
+  label: string;
+  onClick: () => void;
+  disabled?: boolean;
+  iconSize?: number | 'small' | 'medium' | 'large' | 'extraLarge';
+}) => {
+  return (
+    <ButtonText
+      onClick={onClick}
+      css={`
+        padding-left: 0px;
+        &:disabled {
+          .icon-add {
+            opacity: 0.35;
+          }
+          pointer-events: none;
+        }
+      `}
+      disabled={disabled}
+    >
+      <AddIcon
+        className="icon-add"
+        size={iconSize}
+        css={`
+          margin-right: 3px;
+        `}
+        disabled={disabled}
```

Comment:
This `disabled` ends up being added on `<span>` which has no effect. I think it's a mistake made when the new `Icon` component was introduced. We should be able to safely remove `disabled` from here.

Reply:
I just verified this. You are correct. Thanks for taking a deeper look :)
https://github.com/gravitational/teleport/pull/36103/commits/a7b8d57c2652a1f85d96049f3eeb2e726acf607c